walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,186 @@
 
 package com.google.sps;
 
```

Comment:
I think it might be worthwhile to move these two files to your portfolio directory, and get the tests working there.  That way you wouldn't be modifying the walkthroughs, and hopefully it'll help when you need to add tests to the capstone project